src/stan/io/deserializer.hpp

```diff
@@ -606,6 +605,32 @@ class deserializer {
         this->read<Ret>(size - 1), lp);
   }
 
+  /**
+   * Return the next zero sum matrix of the specified dimensions (requires N-1 *
+   * M-1 unconstrained scalars), incrementing the specified reference with the
+   * log absolute Jacobian determinant (no adjustment, in this case).
+   *
+   * <p>See <code>stan::math::sum_to_zero_constrain(Eigen::Matrix,T&)</code>.
+   *
+   * @tparam Ret The type to return.
+   * @tparam Jacobian Whether to increment the log of the absolute Jacobian
+   * determinant of the transform.
+   * @tparam LP Type of log probability.
+   * @param lp The reference to the variable holding the log
```

Comment:
holding the log -> holding the log density

This is tricky terminologically.  A "log probability function" can be a pdf, in which case it returns a density (units of change in probability) as opposed to a probability, or a pdf, in which case it does return probabilities.  But what we don't have in Stan is `target` holding a log probability---it's always a density.  Our doc is a mess around all this.